frontend/components/customizer/path-builder.tsx

```diff
@@ -91,15 +91,15 @@ export function constructPath({
       if (typeof token === 'string') {
         return token.trim()
       } else {
-        const { delimiter, name, optional } = token
+        const { prefix, name, modifier } = token
         const value = namedParams[name]
         if (value) {
-          return `${delimiter}${value.trim()}`
-        } else if (optional) {
+          return `${prefix}${value.trim()}`
+        } else if (modifier === '?' || modifier === '*') {
```

Comment:
The `optional` key has been dropped from the interface, so we're forced to do this check ourselves against the new `modifer` key (which is useful for us in other contexts)